seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/java/org/apache/seatunnel/engine/e2e/RestApiIT.java

```diff
@@ -61,6 +63,7 @@ void beforeClass() throws Exception {
         SeaTunnelConfig seaTunnelConfig = ConfigProvider.locateAndGetSeaTunnelConfig();
         seaTunnelConfig.getHazelcastConfig().setClusterName(testClusterName);
         hazelcastInstance = SeaTunnelServerStarter.createHazelcastInstance(seaTunnelConfig);
+        Common.setDeployMode(DeployMode.CLIENT);
```

Comment:
Actually you don't need set the deployMode, because it will auto set to `CLIENT` if the deployMode is null.

https://github.com/apache/seatunnel/blob/dev/seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/rest/RestHttpPostCommandProcessor.java#L101

Reply:
But without this, the test case will failed now. Could you investigate it?

Reply:
These are failed action. https://github.com/Hisoka-X/seatunnel-fork/actions/runs/6741940408/job/18327335813

Reply:
I am not clear if this is caused by we have some UT case set the DeployMode but doesn't clear this after the ut finished.

Reply:
Let's merge this first. If @liunaijie have better idea for `RestApiIT`. You can create a PR for it.